041-Pandigital_prime.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+'''
+We shall say that an n-digit number is  if it makes use of all the digits
+1 to n exactly once. For example, 2143 is a 4-digit pandigital and is also prime.
+
+What is the largest n-digit pandigital prime that exists?
+'''
+import itertools
+
+def pandigital_numbers(n):
+    # Generate list of number 1 to n.  ex: '1234'
+    list_num = ''.join([str(i) for i in range(1, n + 1)])
+
+    # List all pandigital numbers of digits 1 to n
+    pandigital_str = [''.join(item) for item in itertools.permutations(list_num)]
+    # transfer pendigital number from string to int
+    pandigital_num = [int(num) for num in pandigital_str]
```

Comment:
This line is unnecessary, why not just add in() to the pandigital_str above?

Reply:
I can't join int nums, only string is possible

Reply:
''.join(item) is a string, you can int it.